web/views/pages/index.ejs

```diff
@@ -101,25 +101,23 @@ co2Sub = function(str) {
     <body lang="<%= locale %>">
         <div id="main">
             <div id="header">
-                <div id="header-content">
+                <div id="header-content" class="brightmode">
                     <div class="logo">
-                        <img src="images/icon_noborder_small.png" />
-                        <b>Electricity Map</b> &nbsp;<span style="color: orange">Live</span>
+                        <div class="image" id="electricitymap-logo"></div>
+                        <span class="live" style="font-weight: bold">Live</span>
                         <span class="maintitle small-screen-hidden">
                              · <a href="https://data.electricitymap.org?utm_source=electricitymap.org&utm_medium=referral">Database</a>
                              · <a href="https://api.electricitymap.org?utm_source=electricitymap.org&utm_medium=referral">API</a>
                         </span>
                     </div>
-
-                    <div class="center-header small-screen-hidden">
+                    <!-- <div class="center-header small-screen-hidden">
```

Comment:
commented code. Probably a good idea to remove associated styles / translations if removing.